README.md

```diff
@@ -7,39 +7,45 @@
 
 ## Screenshot
 
+![Screenshot from 2022-03-16 15-10-02](https://user-images.githubusercontent.com/88809610/158707611-ab751c86-6740-4f92-99d3-40ac86214c08.png)
 
 ## Built With
 
 - HTML, CSS, JavaScript
 - Technologies used: GitHub, VScode, nodejs, React, GIT.
 
 
-
-## How it works
-
-
-
-
 ## Live Demo
 
-[Live Demo Link]()
-
+Click [here](https://forhemer.github.io/Math-Magicians/) to see the live demo.
 
 ## Getting Started
-
-
+To get a local copy up and running follow these simple example steps:
 
 ### Prerequisites
 
+- A device that support running browser applications.
+- access to internet connection.
 
 ### Setup
 
 #### View pages from the browser
 
+- If you just want to check the webpage/webapp, you have it available on the live demo link.
 
 
 #### Install locally the code of the application
 
+In order to install a local version of this project and please do the following steps:
+- Install the latest version of any code editor.
+- install the latest version of GIT.
+- open your terminal
+- Go to the [repository](https://github.com/ForHemer/Math-Magicians)  and clone it with `git clone git@github.com:ForHemer/Math-Magicians.git` .
+- open the repository cloned with `cd Math-Magicians`
+- if your code editor is VsCode type `code .`
+- Install npm dependencies with `npm install`
+- To run it type `nmp start`
```

Comment:
- [ ] (Optional) There is a typo in the command for starting the server. You can decide to accept my suggestion and it would automatically be corrected.

```suggestion
- To run it type `npm start`
```